persistence/ravendb/cluster-configuration.md

```diff
@@ -5,11 +5,11 @@ reviewed: 2021-04-12
 versions: '[6.0,)'
 ---
 
-DANGER: Not using cluster-wide transactions in the current form for business data could result in data loss that is impossible to detect. Due to the risk data loss, RavenDB clusters are currently not supported.
+DANGER: Running in cluster-configurations without the use of cluster-wide transactions for business data could result in data loss that is impossible to detect. Due to the risk data loss, RavenDB clusters are currently not supported.
 
 RavenDB version 4 and higher supports configurations with multiple nodes. Support for configurations with a single leader and multiple watchers was originally introduced under the assumption that it’s safe to run without cluster-wide transactions. However, it was discovered that all databases on RavenDB nodes within a cluster behave as primary nodes. That means that any database in a cluster can accept writes. Committing any data safely requires that the data be successfully persisted across a majority of the nodes within the cluster. Optimistic concurrency control cannot be used in cluster scenarios and the only way to enforce this is by using cluster-wide transactions.
```

Comment:
What about something like this:

```suggestion
RavenDB version 4 and higher supports cluster configurations with multiple cluster nodes. Database nodes of a database group distributed across multiple cluster nodes are all operating in master mode, that means that any database node in a cluster can accept writes. Due to missing optimistic concurrency control for cluster-wide transactions, unintended data loss might happen on concurrent writes to different database nodes. Therefore, NServiceBus.RavenDB currently does not support cluster configurations yet.
```

Reply:
Committed a slightly modified version of the suggestion.